infrastructure/backups/download.sh

```diff
@@ -106,7 +106,7 @@ for BACKUP_DIR in /data/backups/*; do
 done
 
 
-mv $BACKUP_RAW_FILES_DIR/extract/elasticsearch /data/backups/elasticsearch
+mv $BACKUP_RAW_FILES_DIR/extract/elasticsearch/* $BACKUP_RAW_FILES_DIR/extract/elasticsearch/.* /data/backups/elasticsearch/
```

Comment:
```suggestion
mv $BACKUP_RAW_FILES_DIR/extract/elasticsearch/* /data/backups/elasticsearch/
```
This is probably what we want right?

Reply:
That would skip hidden files (not sure if there any). Wanted to play it safe and move it all

Reply:
Tested the single parameter approach on staging and it works. Commited your suggestion